docs/faq.rst

```diff
@@ -92,3 +92,22 @@ preview" status. However, you can take a look at `this blog post`_ outlining a
 workable "DIY" solution as a stop gap for now.
 
 .. _`this blog post`: https://www.jeffgeerling.com/blog/2019/how-add-integration-tests-ansible-collection-molecule
+
+Does Molecule support monorepos?
+================================
+
+Yes, roles contained in a monorepo with other roles are automatically picked up and the `ANSIBLE_ROLES_PATH` is set accordingly. See `this page`_ for more information
```

Comment:
Perhaps explain what a "monorepo" is for the rest?

Reply:
Valid point though, adding link to the WP article.